lib/reporters/verbose.js

```diff
@@ -245,6 +251,39 @@ class VerboseReporter {
 		}
 	}
 
+	addTestRunning(evt) {
+		console.log('addTestRunning', '🍇');
+		this.runningTestsCount++;
+		if (!this.runningTestFiles.has(evt.testFile)) {
+			this.runningTestFiles.set(evt.testFile, new Set());
+		}
+		this.runningTestFiles.get(evt.testFile).add(evt.title);
+	}
+
+	removeTestRunning(evt) {
+		console.log('removeTestRunning', '🍇');
+		this.runningTestsCount--;
+		this.runningTestFiles.get(evt.testFile).delete(evt.title);
+	}
+
+	/**
+	 * Generates the summary string of tests that are still running
+	 * @returns string
+	 */
+	runningTestSummary() {
+		console.log('runningTestSummary', '🍇');
+		if (this.runningTestsCount === 0) {
+			return 'NO TESTS 🍓';
+		}
+		let smy = ` ${this.runningTestsCount} tests still running`;
```

Comment:
You don't need to add the space. However let's put a `\n` in front and put a colon after `running`.